### PR TITLE
Added a new date and time format

### PR DESCRIPTION
**Pull request by kelvins**
Added a new date and time format that matches the format dd/mm/yyyy hh:mm:ss.

### CODE REVIEW

**Comment by jekku** [comment]
I think that this, my friend, is also ISO 8601. https://en.wikipedia.org/wiki/ISO_8601. This is a duplicate maybe? @lukehaas

File: src/data/index.js [hunk 1]
```diff
@@ -17,6 +17,12 @@ export const patterns = [{
 	tags: "date,time"
 },
 {
+	name:"Date and Time format",
+	regex:/^(0[1-9]|[12][0-9]|3[01])([\/])(0[1-9]|1[012])\2([0-9][0-9][0-9][0-9]) ([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]$/,
```
Comment (jekku) [inline src/data/index.js]: @kelvins , instead, maybe you could allow the use of slashes OR dashes to delimit the day, month, and year in the existing implementation of the ISO 8601 in the list.

**Comment by lukehaas** [comment]
Thanks, but I think it might be best if you just add optional time to the existing date regex.

**Comment by kelvins** [comment]
Thanks @lukehaas I will do that. I created this date/time regex because in Brazil it's common to use this exactly format: dd/mm/yyyy hh:mm:ss. But actually, it makes sense to include the optional time to the existing date regex. I will close this PR, add the optional time and submit a new one.

**Pull request closed without merging**